website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -140,78 +129,136 @@ spec:
+You can either include the `connectInject.transparentProxy.defaultOverwriteProbes` Helm value to your command or add the `consul.hashicorp.com/transparent-proxy-overwrite-probes` Kubernetes annotation to your pod configuration to overwrite health probes.
+
+Refer to [Kubernetes Health Checks in Consul on Kubernetes](/docs/k8s/connect/health) for additional information.
+
+### Dial services across Kubernetes cluster
+
+If your [Consul servers are federated between Kubernetes clusters](/docs/k8s/installation/multi-cluster/kubernetes), 
+then you must configure services in one Kubernetes cluster to explicitly dial a service in the datacenter of another Kubernetes cluster using the 
+[consul.hashicorp.com/connect-service-upstreams](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) annotation. 
+The following example configures the service to dial an upstream service called `my-service` in datacenter `dc2` on port `1234`:
+
+```yaml
+  "consul.hashicorp.com/connect-service-upstreams": "my-service:1234:dc2"
+```
 
-As of Consul-k8s >= `0.26.0` and Consul-helm >= `0.32.0`, a Kubernetes service that selects application pods is required
-for Connect applications, i.e:
+If your Consul cluster is deployed to a [single datacenter spanning multiple Kubernetes clusters](/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s),
+then you must configure services in one Kubernetes cluster to explicitly dial a service in another Kubernetes cluster using the 
+[consul.hashicorp.com/connect-service-upstreams](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) annotation. 
+The following example configures the service to dial an upstream service called `my-service` in another Kubernetes cluster on port `1234`:
+
+```yaml
+"consul.hashicorp.com/connect-service-upstreams": "my-service:1234"
+```
+
+You do not need to configure services to explicitlly dial upstream services if your Consul clusters are connected with a [peering connection](/docs/connect/cluster-peering). 
+
+## Usage
+
+When transparent proxy is enabled, traffic sent to [KubeDNS](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) 
+or Pod IP addresses is redirected through the proxy. You must use a selector to bind Kubernetes Services to Pods as you define Kubernetes Services in the mesh. 
+The Kubernetes Service name must match the Consul service name to use KubeDNS. This is the default behavior unless you have applied the `consul.hashicorp.com/connect-service` 
+Kubernetes annotation to the service pods. The annotation overrides the Consul service name.
+
+Consul configures traffic redirection for each Pod bound to the Kubernetes Service with `iptables` rules. The rules direct all inbound and outbound traffic through an inbound and outbound listener on the sidecar proxy. Consul configures the proxy to route traffic to the appropriate upstream services based on [service
```

Comment:
```suggestion
Consul configures `iptables` rules for each Pod bound to the Kubernetes Service to redirect all inbound and outbound traffic through an inbound and outbound listener on the sidecar proxy. Consul configures the proxy to route traffic to the appropriate upstream services based on [service
```

Reply:
I like the original one better, they almost mean the same thing but traffic redirection is a good term for what we do with transparent proxy.